.github/CHANGELOG.md

```diff
@@ -37,6 +37,9 @@
 
 ### Bug fixes
 
+* The `.github/workflows/tests_lkcpu_python.yml` workflow properly checks out the release or stable version of Lightning-Qubit during the test job.
```

Comment:
```suggestion
* The `.github/workflows/tests_lkcpu_python.yml` workflow properly checks out the release or stable version of LightningQubit during the test job.
```

Reply:
Super nit-pick suggestion.

Reply:
I think we usually write Lightning-Qubit for the product and `LightningQubit` for the class, so I picked the former in the present case. I'm open to revisiting and making the CHANGELOG more uniform though.

Reply:
Yes, indeed, Disregard my comment.